docs/graphene-workload-tutorial/workload/graphene_sgx/build_gsc_workload.sh.template

```diff
@@ -38,29 +38,13 @@ if [ "$GSC_IMAGE_EXISTS" = "yes" ]; then
 fi
 
 # Manifest files
-MANIFEST_FILE_DIR="${TCF_HOME}/tc/graphene/python_worker/graphene_sgx/manifest"
-MANIFEST_FILES="python.manifest
-sh.manifest
-gcc.manifest
-collect2.manifest
-ld.manifest"
-# Generate list of manifest files
-LIST_MANIFEST_FILES=""
-for f in $MANIFEST_FILES
-do
-  FILE_NAME=${MANIFEST_FILE_DIR}/$f
-  if [ ! -f $FILE_NAME ]; then
-    echo "ERROR:Manifest file $FILE_NAME doesn't exist"
-    exit
-  fi
-  LIST_MANIFEST_FILES+=${MANIFEST_FILE_DIR}/$f
-  LIST_MANIFEST_FILES+=" "
-done
-echo $LIST_MANIFEST_FILES
+
+MANIFEST_FILE="${TCF_HOME}tc/graphene/python_worker/graphene_sgx/manifest/avalon.manifest"
+echo $MANIFEST_FILE
```

Comment:
Need to print manifest filename? You could remove if added for debugging.

Reply:
Added for debugging purpose, removed now